### PR TITLE
Documentation, filename, spelling, license, dependency tweaks

### PR DESCRIPTION
**Pull request by bpbond**
This cleans up a couple NOTEs and a few other things

### CODE REVIEW

**Comment by bpbond** [comment]
```
0 errors ✔ | 0 warnings ✔ | 0 notes ✔
```
🙌

**Comment by bpbond** [comment]
@kdorheim Note removal of `dplyr` dependency in bb298baa24595aa63744d4206e923e5b9b71d6a0 -- I couldn't see any reason to keep it.

File: DESCRIPTION [hunk 1]
```diff
@@ -1,11 +1,16 @@
...
+Title: Microbial Explicit Model Comparison
 Version: 0.1.0.999999
-Authors@R: c(person("Kalyn", "Dorheim", ,email = "kalyn.dorheim@pnnl.gov", role = "aut", comment = c(ORCID = "0000-0001-8093-8397")),
-          person("Jianqiu", "Zheng", ,email = "jianqiu.zheng@pnnl.gov", role = c("aut", "cre"), comment = c(ORCID = "0000-0002-1609-9004")))
+Authors@R: c(
+    person("Kalyn", "Dorheim", email = "kalyn.dorheim@pnnl.gov", 
+        role = "aut", comment = c(ORCID = "0000-0001-8093-8397")),
+    person("Jianqiu", "Zheng", email = "jianqiu.zheng@pnnl.gov", 
+        role = c("aut", "cre"), comment = c(ORCID = "0000-0002-1609-9004"))
+    )
```
Comment (kdorheim) [inline DESCRIPTION:10]: @bpbond add you info?

**Review by kdorheim — APPROVED** [review]
@bpbond see one comment

File: DESCRIPTION (hunk 1 above)
Comment (bpbond) [reply DESCRIPTION:10]: Let's hold off on that for now, and decide later. Thank you!

**Pull request merged**